app/adapters/article.js

```diff
@@ -0,0 +1,3 @@
+import ApplicationAdapter from './application';
```

Comment:
Is this file needed?
Isn't the default for adapters to inherit from the ApplicationAdapter?

Reply:
You're right. I tried deleting it and requests depending on the adapter are still working. I'll remove it.

Reply:
Removed!